### DISCUSSION TITLE
Uncaught (in promise) TypeError: pixi_js__WEBPACK_IMPORTED_MODULE_0__.Sprite.from is not a constructor

### DISCUSSION THREAD

**Discussion by ahmetgozcelik** [pixijs/pixijs#10891] [General]
![image](https://github.com/user-attachments/assets/935095ea-3bf4-424a-aacc-a4460e96807e)
import { Sprite, Text, Container } from "pixi.js";

export default class Letter extends Container {
    constructor(character, x, y) {
        super();
        this.character = character;

        // Harf için bir arka plan kutusu ekleyebiliriz
        this.box = new Sprite.from("letterBox");
        this.box.anchor.set(0.5);
        this.addChild(this.box);

        // Harf karakterini temsil eden metin
        this.text = new Text(this.character, { fontFamily: "Arial", fontSize: 36, fill: 0xffffff });
        this.text.anchor.set(0.5);
        this.addChild(this.text);

        // Konumu ayarla
        this.x = x;
        this.y = y;
    }

    // Harfin seçildiğinde ne yapacağını tanımlar
    select() {
        this.box.tint = 0x00ff00; // Seçildiğinde yeşil renkle işaretle
    }

    // Harfin seçimi kaldırıldığında ne yapacağını tanımlar
    deselect() {
        this.box.tint = 0xffffff; // Normal rengine dön
    }
}

**Comment by bigtimebuddy** [comment]
`new Sprite.from` => `Sprite.from` (don't use `new` here)

**Reply by ahmetgozcelik** [reply]
Thanks, it worked. I'm getting a few more errors like this, but I want to solve it myself. I examine these in the documentation, but I cannot notice them in my code. What do you recommend?

**Reply by ahmetgozcelik** [reply]
![image](https://github.com/user-attachments/assets/ada708c8-5bde-4e44-8fe0-4defea01708d)
The image on the screen changed, but the error did not go away. I think I'm missing something fundamental.

**Reply by bigtimebuddy** [reply]
You need to preload `letterBox`. E.g. `Assets.load({ alias: 'letterBox', src: 'path/to/letterBox.png' })`

**Reply by ahmetgozcelik** [reply]
Thanks! Is there any resource you can recommend to solve this type of errors when I encounter them?

**Reply by bigtimebuddy** [reply]
Definitely check out [pixijs.com's tutorials](https://pixijs.com/8.x/tutorials) or [pixijselementals.com](https://www.pixijselementals.com/#introduction)
